Changes.md

```diff
@@ -1,3 +1,12 @@
+1.2.0
+====
+- Dockerfile :
```

Comment:
Add blank line after heading.

Reply:
Funny you should say that - just noticed that too! fixed.